content/swiftui/concepts/views/terms/toggle/toggle.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Toggle'
+Description: 'A switch that can be turned on or off.'
+Subjects:
+  - 'Software Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Toggle'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A **`Toggle`** is switch that can be used to turn something on or off, or to choose between different options.
+
+## Syntax
+
+```pseudo
+Toggle(_ title: StringProtocol, isOn: Binding<Bool>)
```

Comment:
What is the `_` for? (Sorry, I'm not too versed with Swift 😅)

Reply:
means that the parameter title doesn't have to be written when calling the function.

Toggle("Wifi", isOn: ...) instead of Toggle(title: "Wifi", isOn: ...)

Reply:
Ok gotcha! 👍🏻 And this makes sense because this syntax seems similar to the one used in the official Apple Docs for SwiftUI.

I'm still inclined to suggest we omit the `_` for now and say that `title` is an optional parameter in the text on line # 26. But I wanted to get @yangc95's advice how how to go forward with this one. 😄

Reply:
Hey @Dusch4593 check my solution and let's wait for @yangc95 to move forward.

Reply:
@cilippofilia @Dusch4593 yes! the `_` is a placeholder for unnamed parameters in functions. They can also be used in loops, but I was thinking that we can do something like:

`Toggle("Title String", isOn: Binding<Bool>)`

This way, it's clear that to use this view, we'll follow this exact syntax format. I also think it would be good to take all the text below (within the syntax section) and turn it to bullet points:

- Explain the `"Title String"`...
- Explain that what gets passed into `isOn` param  is a binding boolean and what that means (briefly).

I'd also like to note that the `_` does not indicate that the parameter is optional.